lib/std/compress/lzma.zig

```diff
@@ -0,0 +1,36 @@
+const std = @import("../std.zig");
+const Allocator = std.mem.Allocator;
+const FixedBufferStream = std.io.FixedBufferStream;
+
+pub const decode = @import("lzma/decode.zig");
+pub const LzmaParams = decode.lzma.LzmaParams;
+pub const LzmaDecoder = decode.lzma.LzmaDecoder;
+pub const Lzma2Decoder = decode.lzma2.Lzma2Decoder;
+
+pub fn lzmaDecompress(
+    allocator: Allocator,
+    reader: anytype,
+    writer: anytype,
+    options: decode.Options,
+) !void {
+    const params = try LzmaParams.readHeader(reader, options);
+    var decoder = try LzmaDecoder.init(allocator, params, options.memlimit);
+    defer decoder.deinit(allocator);
+    return decoder.decompress(allocator, reader, writer);
+}
+
+pub fn lzma2Decompress(
```

Comment:
Please consider the fully qualified namespaces of public API:

 * std.compress.lzma.decode (OK)
 * std.compress.lzma.LzmaParams
 * std.compress.lzma.LzmaDecoder
 * std.compress.lzma.Lzma2Decoder
 * std.compress.lzma.lzmaDecompress
 * std.compress.lzma.lzma2Decompress

"lzma" should not appear in the fully qualified namespace twice.

I understand you are trying to disambiguate between lzma and lzma2. Perhaps this disambiguation belongs in the earlier namespace, for example:

 * std.compress.lzma.Decoder
 * std.compress.lzma2.Decoder